tools/azsdk-cli/Azure.Sdk.Tools.Cli.Tests/Helpers/LabelHelperTests.cs

```diff
@@ -2,7 +2,6 @@
 using Azure.Sdk.Tools.Cli.Tests.TestHelpers;
 using Azure.Sdk.Tools.Cli.Configuration;
 
-
```

Comment:
There are no meaningful changes to this file; this newline change can be removed.